src/lib/discord/discordWorker.js

```diff
@@ -63,7 +63,10 @@ class Worker {
 		try {
 			await this.setListeners()
 			await this.client.login(this.token)
-			await this.client.user.setStatus('invisible')
+			if (this.token !== this.config.discord.token[0]) {
```

Comment:
Does this work here? I thought activity had to be set in ready?
If it does work here we should potentially do all the sets here and remove the set from ready
It would be better to pass in an object 'discordStatus' with visibility and activity through the constructor then the caller could set these as it loops through creating the objects and this wouldn't need to understand about any other tokens.

Reply:
It was inconsistent. Sometimes I would start it and both the main bot and the worker would have the activity set here and sometimes only the worker bot would have the activity set here, so I added the check. I'll test to see if removing it from the ready.js would make it consistently be set from here.